core/src/main/scala/com/criteo/cuttle/Workflow.scala

```diff
@@ -56,8 +56,14 @@ trait Workflow[S <: Scheduling] {
     }
   }
 
-  private[cuttle] lazy val roots = vertices.filter(v => edges.forall { case (v1, _, _)  => v1 != v })
-  private[cuttle] lazy val leaves = vertices.filter(v => edges.forall { case (_, v2, _) => v2 != v })
+  private[cuttle] lazy val roots = {
```

Comment:
format trailling lines

Reply:
I do not see any trailing line issue except no empty line between the definitions of roots and leaves?

Reply:
that is it